internal/support/scheduler/router.go

```diff
@@ -91,14 +91,16 @@ func metricsHandler(w http.ResponseWriter, _ *http.Request) {
 	runtime.ReadMemStats(&rtm)
 
 	// Miscellaneous memory stats
-	t.Alloc = rtm.Alloc
-	t.TotalAlloc = rtm.TotalAlloc
-	t.Sys = rtm.Sys
-	t.Mallocs = rtm.Mallocs
-	t.Frees = rtm.Frees
+	t.Memory.Alloc = rtm.Alloc
+	t.Memory.TotalAlloc = rtm.TotalAlloc
+	t.Memory.Sys = rtm.Sys
+	t.Memory.Mallocs = rtm.Mallocs
+	t.Memory.Frees = rtm.Frees
 
 	// Live objects = Mallocs - Frees
-	t.LiveObjects = t.Mallocs - t.Frees
+	t.Memory.LiveObjects = t.Memory.Mallocs - t.Memory.Frees
+
+	t.CpuBusyAvg = usageAvg
```

Comment:
Suggest encapsulating all of the above logic into your `telemetry.SystemUsage` type.

```
t := telemetry.NewSystemUsage(usageAvg)
encode(t, w)
```
where NewSystemUsage() contains all of the above gathering/assigning of stats. This logic is the same in all routers, so would be good to consolidate/refactor.